coops-ui/package-lock.json

```diff
@@ -0,0 +1,15337 @@
+{
```

Comment:
This package-lock.json and the other one at lear root should not be part of this PR as far as I can tell (also coops-ui folder has moved since this was checked in).